src/libraries/shims/manual/Directory.Build.props

```diff
@@ -14,16 +14,16 @@
     <BaseIntermediateOutputPath>$([MSBuild]::NormalizeDirectory('$(ArtifactsObjDir)', '$(OutDirName)'))</BaseIntermediateOutputPath>
     <IntermediateOutputPath>$(BaseIntermediateOutputPath)$(TargetFramework)-$(Configuration)</IntermediateOutputPath>
     <!-- Use runtime assemblies for the TargetFramework, using TargetOS -->
-    <RefPath>$(ArtifactsBinDir)runtime/$(BuildTargetFramework)-$(TargetOS)-$(Configuration)-$(ArchGroup)/</RefPath>
+    <ShimRefPath>$(ArtifactsBinDir)runtime/$(BuildTargetFramework)-$(TargetOS)-$(Configuration)-$(ArchGroup)/</ShimRefPath>
```

Comment:
Out of curiosity, why is this rename needed?

Reply:
Earlier the refpath property was in DBP, so we can overwrite it here but after we move it DBT, it was no longer happening.  We might move these to .Targets as well. 
When we address the bigger problem of not using target framework derived properties in DBP